pkg/metrics/metrics.go

```diff
@@ -0,0 +1,110 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	SuccessTrue  = "true"
+	SuccessFalse = "false"
+)
+
+var (
```

Comment:
Could you add some comments about these prom metrics? Would be helpful in future when trying to understand this

Reply:
Sure